Server/src/server/router.ts

```diff
@@ -133,10 +145,42 @@ export class OcppMessageRouter implements IMessageRouter {
         return this._sender.send(_message);
     }
 
-    routeCallError(client: IClientConnection, message: CallError): Promise<IMessageConfirmation> {
+    routeCallError(client: IClientConnection, message: CallError, action: CallAction): Promise<IMessageConfirmation> {
+        let messageId = message[1];
+        let payload = new OcppError(messageId, message[2], message[3], message[4]);
+
+        // TODO: Add tenantId to context
```

Comment:
How do we get the tenantId?

Reply:
not clear, which is why it's not done yet. We would have to make a decision about where and how to store stationId:tenantId mappings.